client/state/user-devices/actions.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Internal dependencies
+ */
+import {
+	USER_DEVICES_REQUEST,
+	USER_DEVICES_ADD,
+} from 'state/action-types';
+
+export const requestUserDevices = () => ( { type: USER_DEVICES_REQUEST } );
```

Comment:
Could we add some jsDoc to both these functions? It's helpful when folks are trying to look for the correct actions.